tests/integration_tests/bank/case.go

```diff
@@ -624,7 +625,7 @@ func tryGetEndTs(db *sql.DB, tableName string) (result string, ok bool) {
 	var line dataRow
 	row := db.QueryRow(query, tableName)
 	if err := row.Scan(&line.JobID, &line.DBName, &line.TblName, &line.JobType, &line.SchemaState, &line.SchemeID,
-		&line.TblID, &line.RowCount, &line.StartTime, &line.EndTime, &line.State); err != nil {
+		&line.TblID, &line.RowCount, &line.CreateTime, &line.StartTime, &line.EndTime, &line.State); err != nil {
```

Comment:
Prefer using the query  `select a,b,c from ddl_jobs` to retrieve the DDL jobs to avoid compatibility breaks.